src/System.Reflection.Metadata/dir.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <PackageVersion>1.4.1</PackageVersion>
+    <PackageVersion>1.4.2</PackageVersion>
     <AssemblyVersion>1.4.1.0</AssemblyVersion>
```

Comment:
But doesn't that mean that VS would have to take a dependency on the updated assembly version? 

What I think we want to have happen is VS updates the package version to 1.4.2 and they get the 1.4.1 assembly version.